### PR TITLE
[DFMD-5] Create & Style the Customer Signup Page

### PR DESCRIPTION
**Pull request by Aarinolaa**
file:///C:/Users/Osagie/Documents/Arinola%20Dufuna/Customersetuppage.html

image

I wasn't sure how to get the screenshot without reducing the screen size. This is at 50% reduction.

### CODE REVIEW

**Comment by Dufuna-Fem** [comment]
Weldone @Aarinolaa Kindly add scrennshots for tablet and mobile screens.

File: submissions/arinolaOsagie/CSS/signup.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<!-- saved from url=(0073)file:///C:/Users/Osagie/Documents/Arinola%20Dufuna/Customersetuppage.html -->
```
Comment (Dufuna-Fem) [inline submissions/arinolaOsagie/CSS/signup.html:2]: remove commented out code

File: submissions/arinolaOsagie/CSS/signup.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
```
Comment (Dufuna-Fem) [inline submissions/arinolaOsagie/CSS/signup.html:4]: The opening head tag is missing

File: submissions/arinolaOsagie/CSS/signup.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            </div>
+        </div>          
+       <div>
+         <p class="Copyright-2020-Yoursitecom-All-rights-reserved"> © 2021 Yoursite.com All Rights Reserved</p>
+        </div>
+ </div>
+</footer>
+
+
+</body></html>
```
Comment (Dufuna-Fem) [inline submissions/arinolaOsagie/CSS/signup.html:98]: place the closing tags on separate lines

**Review by Dufuna-Fem — DISMISSED** [review]
Remove all spaces in your image names

**Comment by Aarinolaa** [comment]
image
image

**Comment by Aarinolaa** [comment]
image
image

**Comment by Ireti203** [comment]
The header doesnt look appropriate on the mobile view. Kindly update this.

**Comment by Ireti203** [comment]
The form is also not properly aligned as seen in the mock up. Kindly make corrections

**Review by Ireti203 — COMMENTED** [review]
Remove the spaces in the image names.

**Comment by Aarinolaa** [comment]
Dear Ireti. I have made the changes.
Regards,
Arinola

**Comment by adaugochi** [comment]
You have two open PRs, I will have to close this for now

**Pull request closed without merging**